components/account/ModalCollection.vue

```diff
@@ -126,13 +185,14 @@
 <style lang="scss" scoped>
   .btn-collection {
```

Comment:
When the button has a background image, the text is kind of hard to read. Suggestion: 

```
diff --git a/components/account/ModalCollection.vue b/components/account/ModalCollection.vue
index ecde5157..3914f637 100644
--- a/components/account/ModalCollection.vue
+++ b/components/account/ModalCollection.vue
@@ -77,7 +77,7 @@
           class="btn-collection w-100 text-left"
           @click="addItem(collection.id)"
         >
-          {{ displayField(collection, 'title') }} ({{ collection.visibility }}) - {{ collection.total }} items
+          <span>{{ displayField(collection, 'title') }} ({{ collection.visibility }}) - {{ collection.total }} items</span>
         </b-button>
       </div>
       <div class="modal-footer">
@@ -188,7 +188,25 @@
     font-weight: 500;
     margin-bottom: 0.5rem;
     padding: 1rem;
+    position: relative;
     text-transform: none;
+
+    &.btn-overlay {
+      span {
+        position: relative;
+        z-index: 10;
+      }
+
+      &:after {
+        content: '';
+        background: rgba(0, 0, 0, 0.7);
+        height: 100%;
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+      }
+    }
   }
 
   .collections {
```

Reply:
Agreed, and applied.